docs/helm/README.md

```diff
@@ -63,28 +69,27 @@ the installation.
 
 ## Deploy
 
-For the best results we recommend [deploying ACS to AWS
-EKS](./eks-deployment.md). If you have a machine with at least 16GB of memory
+For the best results we recommend [deploying ACS to AWS EKS](./eks-deployment.md). If you have a machine with at least 16GB of memory
 you can [deploy using Docker Desktop](./desktop-deployment.md) (or similar apps
 like Rancher and Podman Desktop) or via [KinD](kind-deployment.md) which just
 requires a working Docker install on any OS.
 
 The recommended cluster resources for the Enterprise version with the components enabled by default are:
 at least 3 nodes with 12 cpu cores and 32 GB of memory in total. You can install with lower
 requirements by fine tuning the [resource requests](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes)
-available in the [values](../../helm/alfresco-content-services/values.yaml) for each component.
+available in the [values](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/values.yaml) for each component.
 
-There are also several [examples](./examples) showing how to deploy with various configurations:
+There are also several [examples](../helm-examples.md) showing how to deploy with various configurations:
 
 * [Deploy with AWS Services (S3, RDS and MQ)](./examples/with-aws-services.md)
 * [Deploy with Intelligence Services](./examples/with-ai.md)
 * [Deploy with Microsoft 365 Connector (Office Online Integration)](./examples/with-ooi.md)
 * [Deploy with external Keycloak SSO authentication](./examples/with-keycloak.md)
 * [Deploy with external infrastructure components](./examples/with-external-infrastructure.md) (e.g. elasticsearch, activemq, postgres)
-* [Enable access to Search Services](./examples/search-services.mdi#enable-alfresco-search-services-external-access)
-* [Enable Email Services](./examples/email-enabled.md)
-* [Use a custom metadata keystore](./examples/custom-metadata-keystore.md)
-* [Install ACS license as part of the deployment](./examples/alf_license.md)
+* [Enable access to Search Services](./examples/search-services.md#enable-alfresco-search-services-external-access)
+* [Enable Email Services](https://github.com/Alfresco/alfresco-helm-charts/blob/main/charts/alfresco-repository/docs/email.md)
+* [Use a custom metadata keystore](https://github.com/Alfresco/alfresco-helm-charts/blob/main/charts/alfresco-repository/docs/keystores.md)
+* [Install ACS license as part of the deployment](https://github.com/Alfresco/alfresco-helm-charts/blob/main/charts/alfresco-repository/docs/enterprise-license.md)
```

Comment:
rewrite them to the new docs at https://alfresco.github.io/alfresco-helm-charts/